lib/hydrothermalVenture.ex

```diff
@@ -0,0 +1,42 @@
+defmodule AdventCode2 do
+  def createRange([{a, b}, {a, c}]) do
+    Enum.to_list(b..c)
+    |> Enum.map(fn x -> {a, x} end)
```

Comment:
Мы начинаем пайпы с литерала или переменной. `credo`, например, на это ругнется. Пайпов длиной 1 мы избегаем. А `Range` имплементирует `Enumerable`. Так что все проще:
```suggestion
    Enum.map(b..c, & {a, &1})
```